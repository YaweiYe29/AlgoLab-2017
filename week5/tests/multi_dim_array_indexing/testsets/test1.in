5
2 25 25
0 5
0 5
0 0 22
1 0 79
2 0 94
3 0 95
4 0 38
0 1 95
1 1 46
2 1 65
3 1 27
4 1 36
0 2 40
1 2 11
2 2 41
3 2 14
4 2 56
0 3 94
1 3 1
2 3 28
3 3 3
4 3 4
0 4 79
1 4 51
2 4 46
3 4 27
4 4 62
0 0
1 0
2 0
3 0
4 0
0 1
1 1
2 1
3 1
4 1
0 2
1 2
2 2
3 2
4 2
0 3
1 3
2 3
3 3
4 3
0 4
1 4
2 4
3 4
4 4
2 3 3
0 5
0 5
0 0 56
1 0 2
2 0 90
0 0
1 0
2 0
2 3 5
0 5
0 5
0 0 20
1 0 41
2 0 15
0 0
1 0
2 0
3 0
4 0
1 3 5
0 5
0 13
1 74
2 50
0
1
2
3
4
2 4 5
0 5
0 5
0 0 78
1 0 49
2 0 40
3 0 81
0 0
1 0
2 0
3 0
4 0
