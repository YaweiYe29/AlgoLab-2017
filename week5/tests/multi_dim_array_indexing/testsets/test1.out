22
79
94
95
38
95
46
65
27
36
40
11
41
14
56
94
1
28
3
4
79
51
46
27
62
56
2
90
20
41
15
-1
-1
13
74
50
-1
-1
78
49
40
81
-1
